packages/ndla-i18n/src/index.ts

```diff
@@ -9,6 +9,8 @@
 import IntlProvider from './IntlProvider';
 
 export { default as injectT } from './injectT';
+// @ts-ignore
+export { default as tType } from './t';
```

Comment:
Her klagde typescript på 
```
Cannot re-export a type when the --isolatedModules flag is provided
```
Vi bruker `"isolatedModules": true` for å hindre utilsiktet eksport av typer, løsningen var da å bruke bare en uskyldig ts-ignore her.